assets/src/blocks/Submenu/getHeadingsFromDom.js

```diff
@@ -0,0 +1,26 @@
+import { generateAnchor } from './generateAnchor';
```

Comment:
I didn't make this handle headers that are on the page more than once. We could do that by keeping track of the already generated anchors and add a number at the end.

Another option could be to concatenate the previous level heading anchors. That would be more future proof compared to appending a number. If a heading is added somewhere in between, then old links would be pointing to the wrong section if we're using numbers.